docs/types/custom.md

```diff
@@ -8,7 +8,7 @@ You use a custom class with a classmethod `__get_validators__`. It will be calle
 to get validators to parse and validate the input data.
 
 !!! tip
-    These validators have the same semantics as in [Validators](validators.md), you can
+    These validators have the same semantics as in [Validators](/validators/), you can
```

Comment:
```suggestion
    These validators have the same semantics as in [Validators](../validators.d), you can
```
Better to use paths to md files, in case we end up hosting the docs in a sub-directory like `/latest/userage/validators/`

Reply:
It was faster to start over rather than backtrack link changes.